database/migrations/2024_07_25_102143_change_payment_status_type_in_orders_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('orders', function (Blueprint $table) {
+            $table->string('payment_status')->nullable()->default(null)->change();
```

Comment:
if it's nullable no need to assign default value then

Reply:
The previous migration had a default value of unpaid. When i ran the migration without assigning null as default [ $table->string('payment_status')->nullable()->change() ];, it was still having the default value as unpaid. I searched and got this solution.